HollowKnightComponent.cs

```diff
@@ -949,6 +950,11 @@ enum SplitterAction {
                 case SplitName.TransClaw: shouldSplit = mem.PlayerData<bool>(Offset.hasWallJump) && nextScene != sceneName; break;
                 case SplitName.TransGorgeousHusk: shouldSplit = mem.PlayerData<bool>(Offset.killedGorgeousHusk) && nextScene != sceneName; break;
                 case SplitName.TransDescendingDark: shouldSplit = mem.PlayerData<int>(Offset.quakeLevel) == 2 && nextScene != sceneName; break;
+                case SplitName.TransTear: shouldSplit = mem.PlayerData<bool>(Offset.hasAcidArmour) && nextScene != sceneName; break;
+                case SplitName.TransTearWithGrub: shouldSplit =
+                        mem.PlayerData<bool>(Offset.hasAcidArmour) &&
+                        mem.PlayerDataStringList(Offset.scenesGrubRescued).Contains("Waterways_13"); break;

```

Comment:
I think this got scuffed?
```suggestion
                        mem.PlayerDataStringList(Offset.scenesGrubRescued).Contains("Waterways_13") &&

```